descriptors/ModuleDescriptor-template.json

```diff
@@ -1135,9 +1158,19 @@
         "circulation.requests.instances.item.post",
         "circulation.requests.hold-shelf-clearance-report.get",
         "circulation.inventory.items-in-transit-report.get",
-        "circulation.pick-slips.get"
+        "circulation.pick-slips.get",
+        "circulation.handlers.loan-related-fee-fine-closed.post"
       ]
     },
+    {
+      "permissionName": "circulation.handlers.all",
+      "displayName": "circulation - all subscribers permissions",
+      "description": "Entire set of permissions needed to grant pub/sub user for subscriptions",
+      "subPermissions": [
+        "circulation.handlers.loan-related-fee-fine-closed.post"
+      ],
+      "visible": false
+    },
```

Comment:
Why are the message handler permissions separated from the general permissions?

Reply:
Pub-sub user have to be granted to execute these APIs https://github.com/folio-org/mod-pubsub/pull/69. So i decided to create an "all" permission set instead of granting one by one.

Reply:
Ok, given we don't have any guidance for that, I'll resolve this for the moment